floodviz/utils.py

```diff
@@ -0,0 +1,90 @@
+import requests
+
+"""
+    Takes a valid url and a dictionary of expected column header strings as keys (values do not matter).
+    parse_rdb then requests data from the service at url, and extracts data matching any header values.
+     
+    Data is returned as a list of dict objects, where each object represents a data point. 
+
+"""
+
+
+def parse_rdb(url, header_values):
```

Comment:
I made changes to parse_rdb based on these comments. Let me know what you think when you get a chance. I plan to make the data_column extractor in the near future.